src/plugins/launch/slurm/launch_slurm.c

```diff
@@ -590,6 +590,53 @@ extern int launch_p_create_job_step(srun_job_t *job, bool use_all_cpus,
 	return SLURM_SUCCESS;
 }
 
+static char *_find_quote_token(char *tmp, char *sep, char **last)
```

Comment:
Please document this function.

Reply:
see https://bugs.schedmd.com/show_bug.cgi?id=7658
this commit should be reverted with 19.05.3, so not wasting time to clean up the schedmd code